scss/components/molecules/_alerts.scss

```diff
@@ -0,0 +1,95 @@
+@charset 'UTF-8';
+
+///
+/// ALERTS COMPONENT
+///
+
+// Alert variables
+$decanter-custom-alerts: () !default;
+
+$decanter-alerts: (
+  success: rgba(23, 94, 84, 0.6),   // $color-palo-alto at 60% opacity
+  warning: rgba(234, 171, 0, 0.6),  // $color-sun at 60% opacity
+  error:   rgba(177, 4, 14, 0.5),   // $color-bright-red at 50% opacity
+  info:    rgba(0, 152, 219, 0.6),  // $color-sky at 60% opacity
+);
+
+$alerts: map-merge($decanter-alerts, $decanter-custom-alerts);
```

Comment:
$alerts is a global variable that is too generically named. Instead it would be best to set this to $decanter-alerts with this map merge.